core/src/main/java/com/rim/IceField/Game.java

```diff
@@ -464,12 +471,12 @@ private int getPlayerInfo(PlayerBase player) {
         System.out.print(4 - player.numOfMoves);
         int partsCollected = 0;
         //for every player
-        for (PlayerBase playerBase : players) {
+        for (PlayerBase PlayerBase : players) {
```

Comment:
You should not name the variable with the same name as the defined class, this can cause issues and is not a good practice